### ISSUE TITLE
FTL Vault Allows Preshift Spare Rush

### ISSUE THREAD

**Issue by Kurfursten** [d3athrow/vgstation13#34850]
Kilette demonstrated today that you can observe in the lobby, spawn into the FTL ship before the shift starts, teleport to the bridge with full tools, and grab the spare before anyone can spawn. It's not that hard if you know what you're doing. When you think about it, it was a disastrous mistake to put a cryopod on this vault.

**Comment by jwhitak** [comment]
Cryo pods working before the round starts is to blame tbh

**Comment by angryturnip** [comment]
feature

**Comment by SootyBear42** [comment]
Couldn't this be fixed by making cryopods only useable until after the round's started?

**Comment by Kurfursten** [comment]
Yes, although the fact that anyone can so quickly get a respawn to the station is probably a little concerning in general. It takes a little more effort to wander in as a hobo for example.

**Comment by SootyBear42** [comment]
> Yes, although the fact that anyone can so quickly get a respawn to the station is probably a little concerning in general. It takes a little more effort to wander in as a hobo for example.

Perhaps make a wall around the cryopod that disappears when the round starts,
or if it's not possible to block them the tele could simply be removed.

**Comment by Eneocho** [comment]
feature, don't change it

**Comment by pedr69** [comment]
[feature][wontfix]
